service/create/templates.go

```diff
@@ -137,7 +137,7 @@ USERDATA_FILE={{.MachineType}}
     --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \
     --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \
     --trust-keys-from-https \
-    quay.io/coreos/awscli:025a357f05242fdad6a81e8a6b520098aa65a600 -- aws s3 --region {{.Region}} cp s3://{{.S3DirURI}}/$USERDATA_FILE /var/run/coreos/temp.txt
+    quay.io/coreos/awscli:025a357f05242fdad6a81e8a6b520098aa65a600 -- aws s3 --region {{.Region}} cp s3://{{.S3URI}}/$USERDATA_FILE /var/run/coreos/temp.txt
```

Comment:
Good point about compatibility. As Ross said, this template is already on the nodes, so they won't be affected by the change (also, it's only read at boot, and we don't reboot nodes for now, nor boot new ones), so we're OK.